metrics/src/main/java/com/baeldung/metrics/aspectj/MetricsAspectJMain.java

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.metrics.aspectj;
+
+import com.codahale.metrics.ConsoleReporter;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.SharedMetricRegistries;
+import java.io.PrintStream;
+import java.util.concurrent.TimeUnit;
+
+public class MetricsAspectJMain {
+    static final MetricRegistry registry = new MetricRegistry();
```

Comment:
this should probably be `private`. As a static reference, it should also be called `REGISTRY` or `METER_REGISTRY` as constants should be in uppercase.